---
layout: essay
type: essay
title: Evidence of Scholarly Ability
date: 2020-12-08
labels:
  - PhD Portfolio
---

## Literature Review

[Improving interpretation of microbiome studies by incorporating phylogenetic information](https://carisdak.github.io/essays/literature-review.pdf)

## Publications

### First author
- [Arisdakessian, C.G., Nigro, O.D., Steward, G.F., Poisson, G. and Belcaid, M., 2021. CoCoNet: an efficient deep learning tool for viral metagenome binning. Bioinformatics, 37(18), pp.2803-2810.](https://academic.oup.com/bioinformatics/article/37/18/2803/6211156)
- [Arisdakessian, C., Cleveland, S.B. and Belcaid,
  M., 2020. MetaFlow|mics: Scalable and Reproducible Nextflow
  Pipelines for the Analysis of Microbiome Marker Data. In Practice
  and Experience in Advanced Research Computing (pp. 120-124).](https://dl.acm.org/doi/pdf/10.1145/3311790.3396664)
- [Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX. DeepImpute:
  an accurate, fast, and scalable deep neural network method to impute
  single-cell RNA-seq data. Genome biology. 2019 Dec;20(1):1-4.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1837-6)
  
### Collaborations
- Zhu, X., Wolfgruber, T. K., Tasato, A., Arisdakessian, C., Garmire, D. G., & Garmire, L. X. (2017)
Granatum: a graphical single-cell RNA-Seq analysis pipeline for genomics scientists. Genome medicine
- Du, Y., Huang, Q., Arisdakessian, C., & Garmire, L. X. (2020).
Evaluation of STAR and Kallisto on single cell RNA-Seq data alignment. G3: Genes, Genomes, Genetics
- Garmire DG, Zhu X, Mantravadi A, Huang Q, Yunits B, Liu Y, Wolfgruber T, Poirion O, Zhao T, Arisdakessian C, Stanojevic S. (2021)
GranatumX: A community-engaging, modularized, and flexible webtool for single-cell data analysis. Genomics, Proteomics & Bioinformatics
- Belcaid, M., Arisdakessian, C., & Kravchenko, Y. (2021). 
Efficient DNA sequence partitioning using probabilistic subsets and hypergraphs. In Proceedings of the 36th Annual ACM Symposium on Applied Computing.
- Jani, A.J., Bushell, J., Arisdakessian, C.G. et al. (2021). 
The amphibian microbiome exhibits poor resilience following pathogen-induced disturbance. ISME J
- Spotkaeff, C., Arisdakessian, C., Jungbluth, S., Rappe, M., Steward, G., and Nigro, O. (2021). 
Across the Basement: A Comparison of Viruses in the Crustal Aquifer at North Pond and Juan de Fuca Ridge, vol. 2021
- Belcaid, M., Arisdakessian, C., & Kravchenko, Y. (2022). 
Taming DNA clustering in massive datasets with SLYMFAST. ACM SIGAPP Applied Computing Review
- Nigro, O., Spotkaeff, C. R., Arisdakessian, C., Jungbluth, S., Rappe, M. S., & Steward, G. 
Phylogenomic comparison of viruses across basalt-hosted oceanic basement systems. In 2022 Astrobiology Science Conference. AGU.
- Cleveland, S., Arisdakessian, C., Nelson, C., Belcaid, M., Frank, K., & Jacobs, G. (2022). 
The C-MÄ€IKI Gateway: A Modern Science Platform for Analyzing Microbiome Data. In Practice and Experience in Advanced Research Computing
