<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="title" content="Evidence of Scholarly Ability | Cedric Arisdakessian | Professional Portfolio">
    <meta name="description" content="">
    <meta name="image" content="https://carisdak.github.io/img/profile.jpg">
    <meta property="og:type" content="website">
    <meta property="og:site_name" content="Cedric Arisdakessian | Professional Portfolio">
    <meta property="og:url" content="https://carisdak.github.io/essays/scholarly-ability.html">
    <meta property="og:title" content="Evidence of Scholarly Ability | Cedric Arisdakessian | Professional Portfolio">
    <meta property="og:description" content="">
    <meta property="og:image" content="https://carisdak.github.io/img/profile.jpg">
    <link rel="shortcut icon" href="/favicon.ico">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
    <link rel="stylesheet" href="/css/techfolio-theme/default.css">
    <link rel="stylesheet" type="text/css" href="/css/rouge/github.css">
    <!-- Load MathJax if 'mathjax: true' is found in your _config.yml. -->
    
    <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML">
    </script>
    

    <title>Evidence of Scholarly Ability | Cedric Arisdakessian | Professional Portfolio</title>
  </head>
  <body>
  <header class="navbar navbar-expand navbar-light bg-light bg-gradient border-bottom">
  <div class="container-fluid">
    <a class="navbar-brand" href="/">Cédric Arisdakessian</a>
    <div class="ms-auto">
      <ul class="navbar-nav mb-2 mb-lg-0">
        <a class="nav-link" href="/#projects">Projects</a>
        <a class="nav-link" href="/#essays">Essays</a>
        <a class="nav-link" href="/resume.html">Resume</a>
      </ul>
    </div>
  </div>
</header>

<div class="container py-4">
  <h1 class="display-4">Evidence of Scholarly Ability</h1>
  <span class="date">08 Dec 2020</span>
  <hr>
  <h2 id="literature-review">Literature Review</h2>

<p><a href="https://carisdak.github.io/essays/literature-review.pdf">Improving interpretation of microbiome studies by incorporating phylogenetic information</a></p>

<h2 id="publications">Publications</h2>

<h3 id="first-author">First author</h3>
<ul>
  <li><a href="https://academic.oup.com/bioinformatics/article/37/18/2803/6211156">Arisdakessian, C.G., Nigro, O.D., Steward, G.F., Poisson, G. and Belcaid, M., 2021. CoCoNet: an efficient deep learning tool for viral metagenome binning. Bioinformatics, 37(18), pp.2803-2810.</a></li>
  <li><a href="https://dl.acm.org/doi/pdf/10.1145/3311790.3396664">Arisdakessian, C., Cleveland, S.B. and Belcaid,
M., 2020. MetaFlow|mics: Scalable and Reproducible Nextflow
Pipelines for the Analysis of Microbiome Marker Data. In Practice
and Experience in Advanced Research Computing (pp. 120-124).</a></li>
  <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1837-6">Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX. DeepImpute:
an accurate, fast, and scalable deep neural network method to impute
single-cell RNA-seq data. Genome biology. 2019 Dec;20(1):1-4.</a></li>
</ul>

<h3 id="collaborations">Collaborations</h3>
<ul>
  <li>Zhu, X., Wolfgruber, T. K., Tasato, A., Arisdakessian, C., Garmire, D. G., &amp; Garmire, L. X. (2017)
Granatum: a graphical single-cell RNA-Seq analysis pipeline for genomics scientists. Genome medicine</li>
  <li>Du, Y., Huang, Q., Arisdakessian, C., &amp; Garmire, L. X. (2020).
Evaluation of STAR and Kallisto on single cell RNA-Seq data alignment. G3: Genes, Genomes, Genetics</li>
  <li>Garmire DG, Zhu X, Mantravadi A, Huang Q, Yunits B, Liu Y, Wolfgruber T, Poirion O, Zhao T, Arisdakessian C, Stanojevic S. (2021)
GranatumX: A community-engaging, modularized, and flexible webtool for single-cell data analysis. Genomics, Proteomics &amp; Bioinformatics</li>
  <li>Belcaid, M., Arisdakessian, C., &amp; Kravchenko, Y. (2021). 
Efficient DNA sequence partitioning using probabilistic subsets and hypergraphs. In Proceedings of the 36th Annual ACM Symposium on Applied Computing.</li>
  <li>Jani, A.J., Bushell, J., Arisdakessian, C.G. et al. (2021). 
The amphibian microbiome exhibits poor resilience following pathogen-induced disturbance. ISME J</li>
  <li>Spotkaeff, C., Arisdakessian, C., Jungbluth, S., Rappe, M., Steward, G., and Nigro, O. (2021). 
Across the Basement: A Comparison of Viruses in the Crustal Aquifer at North Pond and Juan de Fuca Ridge, vol. 2021</li>
  <li>Belcaid, M., Arisdakessian, C., &amp; Kravchenko, Y. (2022). 
Taming DNA clustering in massive datasets with SLYMFAST. ACM SIGAPP Applied Computing Review</li>
  <li>Nigro, O., Spotkaeff, C. R., Arisdakessian, C., Jungbluth, S., Rappe, M. S., &amp; Steward, G. 
Phylogenomic comparison of viruses across basalt-hosted oceanic basement systems. In 2022 Astrobiology Science Conference. AGU.</li>
  <li>Cleveland, S., Arisdakessian, C., Nelson, C., Belcaid, M., Frank, K., &amp; Jacobs, G. (2022). 
The C-MĀIKI Gateway: A Modern Science Platform for Analyzing Microbiome Data. In Practice and Experience in Advanced Research Computing</li>
</ul>

</div>

<footer class="navbar navbar-expand navbar-light bg-light bg-gradient border-top">
  <div class="container-fluid">
    <div class="ms-auto">
      <ul class="navbar-nav mb-2 mb-lg-0">
        <small><a class="nav-link" href="https://techfolios.github.io">Made with Techfolios</a></small>
      </ul>
    </div>
  </div>
</footer>


  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
  </body>
</html>
