---
layout: essay
type: essay
title: Evidence of Scholarly Ability
date: 2020-12-08
labels:
  - PhD Portfolio
---

## Literature Review

[Improving interpretation of microbiome studies by incorporating phylogenetic information](https://puumanamana.github.io/essays/literature-review.pdf)

## Publications

### First author
- [Arisdakessian, C., Cleveland, S.B. and Belcaid,
  M., 2020. MetaFlow|mics: Scalable and Reproducible Nextflow
  Pipelines for the Analysis of Microbiome Marker Data. In Practice
  and Experience in Advanced Research Computing (pp. 120-124).](https://dl.acm.org/doi/pdf/10.1145/3311790.3396664)
- [Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX. DeepImpute:
  an accurate, fast, and scalable deep neural network method to impute
  single-cell RNA-seq data. Genome biology. 2019 Dec;20(1):1-4.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1837-6)
  
### Collaborations
- Jani A., Bushell J., Arisdakessian C., Belcaid M., Boiano D., Brown
  C., Knapp R., 2020. The amphibian microbiome exhibits poor resilience
  following pathogen-induced disturbance  The ISME journal (accepted
  manuscript)
- [Du, Y., Huang, Q., Arisdakessian, C. and Garmire,
  L.X., 2020. Evaluation of STAR and Kallisto on Single Cell RNA-Seq
  Data Alignment. G3: Genes, Genomes, Genetics, 10(5), pp.1775-1783.](https://www.g3journal.org/content/ggg/10/5/1775.full.pdf)
- [Zhu, X., Yunits, B., Wolfgruber, T., Liu, Y., Huang, Q., Poirion,
  O., Arisdakessian, C., Zhao, T., Garmire, D. and Garmire,
  L., 2019. GranatumX: A community engaging and flexible software
  environment for single-cell analysis. bioRxiv, p.385591.](https://www.biorxiv.org/content/10.1101/385591v2.full.pdf)
- [Zhu, X., Wolfgruber, T.K., Tasato, A., Arisdakessian, C., Garmire,
  D.G. and Garmire, L.X., 2017. Granatum: a graphical single-cell
  RNA-Seq analysis pipeline for genomics scientists. Genome medicine,
  9(1), pp.1-12.](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0492-3?utm_campaign=BMCF_TrendMD_2020_GenomeMedicine&utm_source=TrendMD&utm_medium=cpc)

### Accepted manuscripts
- Belcaid M., Arisdakessian C., Kravchenko Y. 2020. Efficient DNA
  Sequence Partitioning Using Probabilistic Subsets and Hypergraphs

### Manuscripts in review
- Arisdakessian C, Nigro O., Steward G., Poisson G., Belcaid
  M. 2020. CoCoNet: An Efficient Deep Learning Tool for Viral Metagenome
  Binning
  
